### PR TITLE
develop

### PR DESCRIPTION
**Pull request by Rukeeo1**
Hello. I just created a pull request. Waiting for your review. Thanks
Screenshot 2019-05-08 at 9 23 54 AM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 node_modules
```
Comment (mogbeyi-david) [inline .gitignore]: Please name the repo properly

File: helpers/helper.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+//converts strings to uppercase using unicode characters...used by toUpper method.
+function toUpperCasing(str) {
+  let newStr = "";
```
Comment (mogbeyi-david) [inline helpers/helper.js]: Use single quotes for Javascript strings

File: string_prototype.js [hunk 3]
```diff
@@ -1,3 +1,90 @@
...
+};
+
+
+String.prototype.ucFirstAgain = function(){
+  //split string
+  let stringToArray = this.split(' ');
+  return stringToArray.map((item) => item.ucFirst()).join(' ');
+}
+
+console.log('rukee ojigbo'.ucFirstAgain());
```
Comment (mogbeyi-david) [inline string_prototype.js]: Remove console.log statements

File: tests/string_prototype.test.js [hunk 4]
```diff
@@ -0,0 +1,230 @@
+const String = require("../string_prototype");
```
Comment (mogbeyi-david) [inline tests/string_prototype.test.js]: Attach a snippet of your code coverage

File: helpers/helper.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+  return newString.replace(regex, "");
+}
+
+function strNumToWord(str) {
+  let newString = "";
+  let regex = /\s/;
+
+  for (var i = 0; i < str.length; i++) {
+    //pass in the the number index into the array to return the corresponding word format.
```
Comment (mogbeyi-david) [inline helpers/helper.js]: Grammatical error in this comment

File: string_prototype.js [hunk 6]
```diff
@@ -1,3 +1,90 @@
...
+String.prototype.words = function() {
+  let regex = /[a-zA-Z]+/g;
+  return this.match(regex);
+};
+
+String.prototype.wordCount = function() {
+  return this.words().length;
+};
+
+//toCurrency ....... implementation comming soon.
```
Comment (mogbeyi-david) [inline string_prototype.js]: uhmmm...what?!

File: string_prototype.js [hunk 7]
```diff
@@ -1,3 +1,90 @@
...
+
+String.prototype.toCurrency = function(number) {
+  return number.toFixed(2).replace(/\d(?=(\d{3})+\.)/g, "$&,");
+};
+
+
+String.prototype.ucFirstAgain = function(){
+  //split string
+  let stringToArray = this.split(' ');
+  return stringToArray.map((item) => item.ucFirst()).join(' ');
```
Comment (mogbeyi-david) [inline string_prototype.js]: This line needs a comment to explain what it is doing

File: string_prototype.js (hunk 6 above)
Comment (Rukeeo1) [reply string_prototype.js]: that was an initial comment while i was still working.  it has since been removed

File: string_prototype.js (hunk 3 above)
Comment (Rukeeo1) [reply string_prototype.js]: I have converted to single quotes for strings my latest version

Comment (Rukeeo1) [reply string_prototype.js]: I have removed console.log from the latest version

File: helpers/helper.js (hunk 5 above)
Comment (Rukeeo1) [reply helpers/helper.js]: i have rectified the grammatical error

**Pull request merged**